Library/Formula/strider.rb

```diff
@@ -0,0 +1,93 @@
+class UniversalNpm < Requirement
+  fatal true
+  satisfy { which("npm") }
+  def message
+    "npm is required. If you have installed node with `--without-npm` option, reinstall with `--with-npm`."
```

Comment:
Likewise, not sure why this cant be ` depends_on "node" => "with-npm" `. One of the maintainers can correct me if desired.